.gitignore

```diff
@@ -17,6 +17,9 @@
 /.next/
 /out/
 
+# development
+/_site
```

Comment:
I don't see this directory. Where is that coming from?

Reply:
Ah you know what that's a holdover from the previous architecture. Removing.